### ISSUE TITLE
Using Space key on the play/pause button in Firefox does not work

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#2888]
Since moving to V5, hitting Enter when the Play/pause button has focus in Firefox acts as expected, and using Enter or Space on Chrome works, but using the Space key on the Play/Pause button with Firefox does not work. It acts a little bit like the button is pressed twice, briefly playing or pausing the video and then switching back to playing or pausing, whichever was being done before the key press.

I was trying to track down if this is a duplicate of an issue which was already reported, and can't find one that's identical. Potentially related to #2669, though.

**Comment by gkatsev** [comment]
Fixes by #3032.
